ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/AddRemoveEditPanel.java

```diff
@@ -0,0 +1,34 @@
+package com.drewhannay.chesscrafter.panel;
+
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.DefaultListModel;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.ListCellRenderer;
+import java.awt.Dimension;
+
+public class AddRemoveEditPanel extends JPanel {
```

Comment:
Also, you can do this:
`public class AddRemoveEditPanel<T> extends ChessPanel`
and then you can get rid of the `<T>` at the beginning of the constructor. You generally only want a generic method if it's some utility method, otherwise make the whole class generic